test/utils.go

```diff
@@ -54,16 +57,20 @@ const aerospikeConfigSecret string = "aerospike-config-secret" //nolint:gosec //
 
 var aerospikeVolumeInitMethodDeleteFiles = asdbv1beta1.AerospikeVolumeMethodDeleteFiles
 
-func initConfigSecret(secretDir string) error {
-	secrets = make(map[string][]byte)
+func initConfigSecret(secretDirectory string) error {
```

Comment:
IMO, it's better to return the populated map from here and assign it to respective `secrets` and `cacertSecrets` var instead of adding multiple if and else condition for better readability